packages/runner/injection/patches/fetch.ts

```diff
@@ -34,17 +34,13 @@ export const patchFetch = (Cypress: Cypress.Cypress, window) => {
       credentials = credentials || 'same-origin'
       // if the option is specified, communicate it to the the server to the proxy can make the request aware if it needs to potentially apply cross origin cookies
       // if the option isn't set, we can imply the default as we know the resource type in the proxy
-      if (url) {
-        // @ts-expect-error
-        Cypress.backend('request:sent:with:credentials', {
-          // TODO: might need to go off more information here or at least make collisions less likely
-          url,
-          resourceType: 'fetch',
-          credentialStatus: credentials,
-        })
-      }
+      await requestSentWithCredentials({
```

Comment:
this function handles the url and checks if cypress is on window, this is common between fetch and xhr calls.